spring-batch-elasticsearch/pom.xml

```diff
@@ -1,62 +1,92 @@
+	</parent>
+
 	<groupId>org.springframework.batch.extensions</groupId>
 	<artifactId>spring-batch-elasticsearch</artifactId>
 	<version>0.1.0-SNAPSHOT</version>
 
+	<name>Spring Batch Elasticsearch</name>
+	<description>Spring Batch extension for Elasticsearch</description>
+	<url>https://github.com/spring-projects/spring-batch-extensions/tree/main/spring-batch-elasticsearch</url>
+	<inceptionYear>2014</inceptionYear>
+	<licenses>
+		<license>
+			<name>Apache-2.0</name>
+			<url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+			<distribution>repo</distribution>
+		</license>
+	</licenses>
+
+	<scm>
+		<url>https://github.com/spring-projects/spring-batch-extensions/tree/main/spring-batch-elasticsearch</url>
+	</scm>
+
 	<properties>
-		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<java.version>1.8</java.version>
-		<spring-batch.version>3.0.10.RELEASE</spring-batch.version>
-		<spring-data-elasticsearch.version>1.0.1.RELEASE</spring-data-elasticsearch.version>
-		<mockito.verion>1.9.5</mockito.verion>
-		<junit.verion>4.13.1</junit.verion>
+		<!-- Dependency versions overriding -->
+		<junit.version>4.13.1</junit.version>
 	</properties>
 
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-compiler-plugin</artifactId>
-				<version>2.5.1</version>
-				<configuration>
-					<source>${java.version}</source>
-					<target>${java.version}</target>
-				</configuration>
-			</plugin>
-		</plugins>
-	</build>
-
 	<dependencies>
-	
+		<!-- Compile -->
 		<dependency>
 			<groupId>org.springframework.batch</groupId>
 			<artifactId>spring-batch-core</artifactId>
-			<version>${spring-batch.version}</version>
 		</dependency>
-
 		<dependency>
 			<groupId>org.springframework.data</groupId>
 			<artifactId>spring-data-elasticsearch</artifactId>
-			<version>${spring-data-elasticsearch.version}</version>
 		</dependency>
-
+		<!-- Test -->
 		<dependency>
-			<groupId>org.mockito</groupId>
-			<artifactId>mockito-all</artifactId>
-			<version>${mockito.verion}</version>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
 			<scope>test</scope>
 		</dependency>
-
 		<dependency>
-			<groupId>junit</groupId>
-			<artifactId>junit</artifactId>
-			<version>${junit.verion}</version>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-core</artifactId>
 			<scope>test</scope>
 		</dependency>
-
 	</dependencies>
 
-</project>
\ No newline at end of file
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>flatten-maven-plugin</artifactId>
+				<version>1.6.0</version>
+				<executions>
+					<execution>
+						<id>flatten</id>
+						<phase>process-resources</phase>
+						<goals>
+							<goal>flatten</goal>
+						</goals>
+						<configuration>
+							<flattenMode>ossrh</flattenMode>
+							<pomElements>
+								<developers>remove</developers> <!-- FIXME remove once the developers are defined -->
```

Comment:
@fmbenhassine I understand from https://github.com/spring-projects/spring-batch-extensions/pull/166#issuecomment-2575075572 that @hpoettker will maintain this module.

As I couldn't add their handle as a reviewer, I assume the onboarding is not yet finished. So, I haven't added any `developers` section to the POM and added this parameter to prevent pulling the ones defined in the `spring-boot-starter-parent`.